protocol/x/vault/client/cli/tx.go

```diff
@@ -18,5 +23,67 @@ func GetTxCmd() *cobra.Command {
 		SuggestionsMinimumDistance: 2,
 		RunE:                       client.ValidateCmd,
 	}
+
+	cmd.AddCommand(CmdDepositToVault())
+
+	return cmd
+}
+
+func CmdDepositToVault() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deposit-to-vault [vault_type] [vault_number] [depositor_owner] [depositor_number] [quantums]",
+		Short: "Broadcast message DepositToVault",
+		Args:  cobra.ExactArgs(5),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			// Parse vault type.
+			vaultType, err := GetVaultTypeFromString(args[0])
+			if err != nil {
+				return err
+			}
+
+			// Parse vault number.
+			vaultNumber, err := strconv.ParseUint(args[1], 10, 32)
```

Comment:
nit: Use these [functions](https://github.com/spf13/cast/blob/master/caste.go#L588) instead to cast?

Reply:
good call. updated!